languages/tree-sitter-stack-graphs-typescript/rust/lib.rs

```diff
@@ -9,16 +20,28 @@ const STACK_GRAPHS_BUILTINS_CONFIG: &str = include_str!("../src/builtins.cfg");
 /// The stack graphs builtins source for this language
 const STACK_GRAPHS_BUILTINS_SOURCE: &str = include_str!("../src/builtins.ts");
 
-pub fn language_configuration() -> LanguageConfiguration {
+pub fn language_configuration(cancellation_flag: &dyn CancellationFlag) -> LanguageConfiguration {
+    let language = tree_sitter_typescript::language_typescript();
+    let sgl = StackGraphLanguage::from_str(language, STACK_GRAPHS_TSG_SOURCE).unwrap();
+    let mut builtins = StackGraph::new();
+    let file = builtins.add_file("<builtins>").unwrap();
+    let mut builtins_globals = Variables::new();
+    Loader::load_globals_from_config_str(STACK_GRAPHS_BUILTINS_CONFIG, &mut builtins_globals)
+        .unwrap();
+    sgl.build_stack_graph_into(
+        &mut builtins,
+        file,
+        STACK_GRAPHS_BUILTINS_SOURCE,
+        &builtins_globals,
+        cancellation_flag,
+    )
+    .unwrap();
```

Comment:
This is a fair bit of boilerplate that we'll have to copy/paste for each language.  Is there any part of it that we can factor our into a helper method in `tree-sitter-stack-graphs`?  (Might be something we hold off on doing until we have >1 language so it's easier to see what parts are genuinely in common and could be factored out)

Reply:
I think that is a good idea! I think we can do that now.